README.md

```diff
@@ -9,88 +9,57 @@
-    1. If the reason you can't write cross platform code is because there is a bug in ReactNative that is preventing it from working, the correct action is to fix RN and submit a PR upstream -- not to hack around RN bugs with platform-specific code paths.
-    1. If there is a feature that simply doesn't exist on all platforms and thus doesn't exist in RN, rather than doing if (platform=iOS) { }, instead write a "shim" library that is implemented with NOOPs on the other platforms.  For example, rather than injecting platform-specific multi-tab code (which can only work on browsers, because it's the only platform with multiple tabs), write a TabManager class that just is NOOP for non-browser platforms.  This encapsulates the platform-specific code into a platform library, rather than sprinkling through the business logic.
-    1. Put all platform specific code in dedicated files and folders, like /platform, and reject any PR that attempts to put platform-specific code anywhere else.  This maintains a strict separation between business logic and platform code.
+#### Table of contents
+* [Local Development](#local-development)
+* [Running The Tests](#running-the-tests)
+* [Debugging](#debugging)
+* [Structure of the app](#structure-of-the-app)
+* [Philosophy](#Philosophy)
+* [Deploying](#deploying)
+
+#### Additional Reading
+* [Contributing to Expensify.cash](./CONTRIBUTING.md)
+* [Expensify Code of Conduct](./CODE_OF_CONDUCT.md)
+* [Contributor License Agreement](./CLA.md)
 
 ----
 
 # Local development
+These instructions should get you set up ready to work on Expensify.cash 🙌
 
-These instructions should get you set up ready to work on the front-end code of Expensify.
-
-**Note:** Expensify engineers please refer to [these additional instructions](https://stackoverflow.com/c/expensify/questions/7699/7700) for testing against the local API.
-
-## Getting Started
+## Getting Started 
 1. Install `node` & `npm`: `brew install node`
 2. Install `watchman`: `brew install watchman`
 3. Install dependencies: `npm install`
-4. Run `cp .env.production .env` and edit `.env` to set up your local config options.
 
 You can use any IDE or code editing tool for developing on any platform. Use your favorite!
 
 ## Running the web app 🕸
-Contributors who don't have full-access to Expensify's development environment will need to run the app against the production API.
-* In the `.env` file set the `USE_WEB_PROXY` environment variable to `true` to indicate the proxy should be used
-* To run the web app, run the **Development Server**: `npm run proxy`
+* To run the **development web app**: `npm run web`
 * Changes applied to Javascript will be applied automatically via WebPack as configured in `webpack.dev.js`
 
 ## Running the iOS app 📱
 * To install the iOS dependencies, run: `npm install && cd ios/ && pod install`
```

Comment:
I read this line and thought it be a nice addition to have this so they end up back where they started for the next command. Optionally, we could add an npm script like `npm run iosinstall`
```suggestion
* To install the iOS dependencies, run: `npm install && cd ios/ && pod install && cd ..`
```